modules/rostests/apitests/shell32/GetDisplayNameOf.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         LGPLv2.1+ - See COPYING.LIB in the top level directory
+ * PURPOSE:         Test for GetDisplayNameOf
+ * COPYRIGHT:       Copyright 2023 Mark Jansen <mark.jansen@reactos.org>
+ *                  Copyright 2023 Doug Lyons <douglyons@douglyons.com>
+ */
+
+#include "shelltest.h"
+#include <shellutils.h>
```

Comment:
As done here: https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/apitests/shell32/ShellState.cpp;hb=d70e798977260ac74074deb124eb87777cd88914#l11
the fix for the build error should either be:
```suggestion
#include <stdio.h>
#include <shellutils.h>
```
or maybe better (due to C++),
```suggestion
#include <cstdio>
#include <shellutils.h>
```

Reply:
@HBelusca, Thanks very much for your suggestions. Since this built for me locally using MSVC 2017, I was somewhat lost in what direction to go.

Reply:
Sorry, the builders still failed with your second suggestion as shown here:
https://github.com/Doug-Lyons/reactos/actions/runs/5697320183
Now trying with your first suggestion. Thanks.

Reply:
First suggestion built correctly. Thanks.

Reply:
> (greater than and less than not shown by github, so removed)

Use code blocks in the comment (enclosing with one backquote on the left and right for a single line, or using three backquotes at the beginning, and at the end -- both on their own lines -- for a code block)

Reply:
Thanks. Later I realized that is what @JoachimHenze had done above which worked.